ansible-satellite6/roles/partition-disk/tasks/main.yml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
Is it intentional to have roles nested inside a nested ansible-satellite6 directory?

Reply:
Yes, it was intentional.